quodlibet/quodlibet/browsers/paned/main.py

```diff
@@ -227,7 +230,11 @@ def refresh_panes(self):
             tags = [t for t in p.tags if not t.startswith("~#")]
             self.__star.update(dict.fromkeys(tags))
 
-        self.set_wide_mode(config.getboolean("browsers", "pane_wide_mode"))
+        try:
+            mode = config.gettext("browsers", "pane_mode")
+        except:
+            mode = ColumnModes.SMALL
```

Comment:
You can add a default argument to gettext (the third argument), which will be returned if the value is not found